query-engine/schema/src/build/mutations/create_many.rs

```diff
@@ -22,6 +22,50 @@ pub(crate) fn create_many(ctx: &'_ QuerySchema, model: Model) -> OutputField<'_>
     )
 }
 
+/// Builds a create many mutation field (e.g. createManyUsers) for given model.
+pub(crate) fn create_many_and_return(ctx: &'_ QuerySchema, model: Model) -> OutputField<'_> {
+    let field_name = format!("createMany{}AndReturn", model.name());
```

Comment:
If I am not mistaken, this is the first operation where we insert model name in the middle, right? Should we follow all other operations and add it in the end?

Reply:
Nope, it's not the first time. This is actually consistent with the `find*OrThrow` variants https://github.com/prisma/prisma-engines/blob/4fa5f88854499b2bb3746eb678b1546e8cca5066/query-engine/schema/src/build/output_types/query_type.rs#L94

Reply:
Beware that none of this matters much anyway anymore since this will be mapped as `prisma.model.createManyAndReturn` anyway in the client